src/plugins/intel_gpu/tests/test_cases/convolution_gpu_test.cpp

```diff
@@ -9083,11 +9084,10 @@ TEST_P(convolution_gpu_onednn, conv_onednn_cases) {
     for (auto& p : network.get_primitives_info())
         std::cerr << p.original_id << " " << p.kernel_id << std::endl;
 
-    auto out_mem = network.get_output("conv_fsv").get_memory();
-    mem_lock<FLOAT16> out_ptr{ out_mem, get_test_stream() };
-    auto out_lay = out_mem->get_layout();
-
-    ASSERT_EQ(out_mem->get_layout().format, format::byxf);
+    auto out_ptr = network.get_output_values<FLOAT16>("conv_fsv");
```

Comment:
Could you explain why it is changed to `get_output_values`?

Reply:
https://github.com/openvinotoolkit/openvino/pull/14738#discussion_r1058041538 As I mentioned before, it is to avoid side-effects related to output nodes. For example, [this](https://github.com/openvinotoolkit/openvino/pull/14837/files#diff-41a32bc38b6fa9c7db5e656050c380c993d424df879bb6a1e2627e7008a7704bR1866) patch is related to the side-effect of output node. Moreover, many other testcases are also using reorder_bfyx as an output node (possibly for the same purpose as above), and there is no reason that convolution node has to be output node in this testcase. Besides, #14860(new PR of #14143) is not WIP now (in review). Therefore I strongly recommend this change for the view of consistency and safety.
https://github.com/openvinotoolkit/openvino/pull/14738#discussion_r1058041916 Because convolution node is not output node anymore. Using get_output_values() is safer (and more readable!) than using memlock because it guarantees correctness of memory data type.

Reply:
Furthermore, original code throws exception: Unable to lock allocation_type::usm_device with write lock_type.